package.json

```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "harp": "harp ",
     "harp:compile": "npm run harp compile styleguide dist",
-    "start": "harp server styleguide",
+    "start": "concurrently 'harp server styleguide' 'browser-sync start --config bs-config.js'",
+    "bs-start": "browser-sync start --config bs-config.js",
```

Comment:
if this is included inside of `start` is this still necessary? I'm okay with keeping it, but if we do decide to keep it, maybe having a `npm harp-start` command makes sense as well, to have the facilities to start one at a time if we need to.  I don't mind either way, I just don't think it makes sense to have a browser-sync only start command, if we don't have one for harp.